src/cli/index.js

```diff
@@ -64,12 +65,22 @@ async function runTests (argParser) {
         selectorTimeout: opts.selectorTimeout
     });
 
+    await testCafe.close();
+
     exit(failed);
 }
 
-async function listBrowsers () {
-    var installations = await getBrowserInstallations();
-    var aliases       = Object.keys(installations);
+async function listBrowsers (providerName) {
+    var provider = BrowserProviderPool.getProvider(providerName);
+
+    if (!provider)
+        throw new GeneralError(MESSAGE.browserProviderNotFound, providerName);
+
+    await BrowserProviderPool.initProvider(providerName);
```

Comment:
Why we need to explicitly init provider? Can't we do it on `BrowserProviderPool.getProvider`? BTW, `BrowserProviderPool` is an object, not ctor, so it should be named in kebab-case